index.html

```diff
@@ -86,98 +86,6 @@ <h2 class="social-media">Let's connect</h2>
 
```

Comment:
According to the project documentation requirements:

![image](https://user-images.githubusercontent.com/57421823/156497235-ffaf6285-2c32-4692-8036-ae540cf3f5d6.png)


- [ ] On your `readme.md`, please kindly add the missing sections "getting started" from the [Microverse readme template](https://github.com/microverseinc/readme-template).  This section explains the steps needed to run the project locally to your repo's visitors.

- [ ] On your `readme.md`, please add the missing section "Acknowledgments" from the [Microverse readme template](https://github.com/microverseinc/readme-template)..

_Note for the future projects:  In the Microverse readme template, all sections are required except for "Tests" and "Live Demo" if not required by the project itself._

That way, your project will be well documented, in a professional way 💪

Reply:
Thank you for your review @Meltrust. Do I really need to add an acknowledgement if there's no one to acknowledge?

Reply:
Please, kindly do so.  

![image](https://user-images.githubusercontent.com/57421823/156500249-cefa0e27-3f4d-4e12-b5ee-1c3cb33c59a3.png)

You can fill the section like in the example, or with whatever/whoever you want.  You could acknowledge Microverse, your coding partner, authors of articles/lessons that helped you, morning meeting partners, standup meeting partners.... You name it.

Just treat it as a business requirement it should be fine. 🙏

Reply:
![Screenshot 2022-03-03 051809](https://user-images.githubusercontent.com/8092327/156501281-2e21a097-1c84-431c-8ec5-534fd15fbb12.png)

Reply:
> ![Screenshot 2022-03-03 051809](https://user-images.githubusercontent.com/8092327/156501281-2e21a097-1c84-431c-8ec5-534fd15fbb12.png)

@Meltrust This is what the README says ⬆️

Reply:
Let me investigate this inquiry with my superiors and I'll be in touch with you in a little while.

Reply:
Hey @dewslyse, I consulted this and I have an answer for you:

_Please, include the section.  This will help you to have a professional repository._  

If you have other questions, please don't hesitate to contact me again.

Reply:
@Meltrust, could you please tag your superior in this thread? I also need a consult with him or her. 
I don't understand why you won't approve my PR based on what seems like optional suggestions.

Reply:
@dewslyse, of course.  Just fill out this form and you will get a [second opinion](https://airtable.com/shrQAqnBwek5a0O0s) for this review.

I hope this is helpful to you.